test/runtests.jl

```diff
@@ -1,16 +1,21 @@
 using Test
+using Statistics
+using DifferentialEquations
```

Comment:
Since `Statistics` and `DifferentialEquations` and some other packages are only being used (I think) in our test suite, we should create a separate test environment (i.e. create a `test/Project.toml`) so that we keep the ClimaLSM dependencies slim.

Reply:
good idea! Ill try and get this set up.

Reply:
OK - I have questions about how this works haha, maybe we can hop on a VS code and figure it out at some point.

It is saying:
```- Run `import Pkg; Pkg.add("ClimaLSM")` to install the ClimaLSM package.
```

ClimaLSM is not registered yet. I can add it via URL and it will pull from main, but couldnt we run into issues, where the version it pulls is not the version we need on our branch we are trying to merge (because we have made modifiations)?